project.sh

```diff
@@ -420,7 +420,7 @@ require-no-focus () {
 lein-lint () {
 	if is-lein;then
 		require-no-focus
-		copy-to-project '.clj-kondo'
+		copy-to-project '.clj-kondo/config.edn'
```

Comment:
.clj-kondo is a directory that can also contain cache folders.

When the project is linted, it used to not copy in the config because the cache folder existed